nx_parallel/utils/chunk.py

```diff
@@ -20,7 +21,8 @@ def cpu_count():
     # Check if we are running under pytest
```

Comment:
todo: remove cpu_count and instead have the get_configs() and a function that returns default configs for testing.